README.md

```diff
@@ -6,4 +6,4 @@ Wrapper around [SPIR-V Cross](https://github.com/KhronosGroup/SPIRV-Cross) gener
 
 This package intends to provide Julia bindings to the C API of SPIR-V Cross, and will suit the needs of those who only want a barebone wrapper. Abstractions, however, are not planned at the moment, though the package contains a few convenience functions to reflect into compiled SPIR-V shaders.
 
-It is currently used in conjunction with [Vulkan.jl](https://github.com/JuliaGPU/Vulkan.jl) (under active development).
+It is currently used in conjunction with [VulkanCore.jl](https://github.com/JuliaGPU/VulkanCore.jl) (under active development).
```

Comment:
I think we can remove what's in parenthesis. It is not really under _active_ development, it's actually usable (since VulkanCore is intended as a thin Clang-generated wrapper, nothing more) and I doubt that many changes would be planned in there.